spec/requests/api/graphql/queries/submission_spec.rb

```diff
@@ -137,6 +137,7 @@
             offers(gravityPartnerId: "#{gravity_partner_id}") {
               id
               state
+              saleLocation
```

Comment:
I'd love to see `saleLocation` land in that last test too, something like this:

```diff
          expect(offer_response).to match(
            {
              'id' => offer.id.to_s,
              'state' => offer.state,
+             'saleLocation' => offer.sale_location,
              'commissionPercentWhole' => offer.commission_percent_whole
            }
          )
```

How does that sound?